components/MarkdownOutput.tsx

```diff
@@ -0,0 +1,47 @@
+import Stack from "@mui/material/Stack";
+import Button from "@mui/material/Button";
+import { useFetch } from "../hooks/useFetch";
+import dynamic from "next/dynamic";
+import copyToClipBoard from "../services/clipboard";
+const MonacoEditor = dynamic(import("@monaco-editor/react"), { ssr: false });
+import ContentCopyIcon from "@mui/icons-material/ContentCopy";
+
+export default function MarkDownOutput() {
+  const { data } = useFetch((state) => state.wikiEntry);
+  const alterWikiEntry = useFetch((state) => state.alterWikiEntry);
+
+  console.log(data);
+  return (
+    <>
+      <Stack
+        sx={{
+          minWidth: {
+            xs: "400px",
+            sm: "600px",
+            md: "900px",
+          },
+        }}
+      >
+        <MonacoEditor
+          theme="vs-dark"
+          height="500px"
+          language="markdown"
+          value={data ? data.result : "## Hello World"}
+          options={{
+            fontSize: 14,
+          }}
+          onChange={async (value) => {
+            console.log(value);
+            alterWikiEntry(value);
+          }}
+        />
+      </Stack>
+      <Button
+        startIcon={<ContentCopyIcon />}
+        onClick={() => copyToClipBoard(data.result)}
+      >
```

Comment:
On my 16:9 1920x1080 monitor on my opened chrome, the Button is cutted, no gud UX here. I assume its due to Macbooks usually have more height in relation to width.

![image](https://github.com/Satttoshi/github-wiki-generator/assets/109807794/4ac894e3-5350-424c-98de-42a560db7834)

Reply:
sure you werent scrolling the vscode? At my machine it has more then enough space at the bottom